modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,472 @@
+
+.. Redpanda Console SA
+
+... Redpanda Console custom role permissions
++
+.Expand necessary permissions
+[%collapsible]
+====
+* `secretmanager.secrets.create`
+* `secretmanager.secrets.delete`
+* `secretmanager.secrets.list`
+* `secretmanager.secrets.update`
+* `secretmanager.versions.add`
+* `secretmanager.versions.destroy`
+* `secretmanager.versions.disable`
+* `secretmanager.versions.enable`
+* `secretmanager.versions.list`
+* `iam.serviceAccounts.getAccessToken`
+
+NOTE: If `iam.serviceAccounts.getAccessToken`` is not added, there will be errors in the Redpanda Console pod log.
+====
+
+... Project bindings
+
+* Redpanda Console custom role
+
+.. Redpanda Connectors SA
+
+... Connectors custom role permissions
+
+* `resourcemanager.projects.get`
+* `secretmanager.versions.access`
+
+... Project bindings
+
+* Connectors custom role
+
+. Because some resources can only be created after the Redpanda ID is known, xref:./create-byoc-cluster-gcp.adoc[create a BYOC cluster] in the https://cloudv2.redpanda.com[Redpanda Cloud UI^] to get the Redpanda ID.
++
+IMPORTANT: On the Network page, select the *Customer-managed* network connection type. Redpanda validates your network, service account, and storage bucket configuration. Click *Next*, but before running the `rpk` command on the Deploy page of the UI, note the redpanda-id (for example, cisld88gfi809ee1qjcg). You finish the cluster deployment in a later step.  
++
+Now use the redpanda-id to bind the service accounts with the following roles:
++
+.. Service account bindings
+
+... Redpanda cluster SA
+
+* Principal: `serviceAccount:<service-project-id>.svc.id.goog[redpanda/rp-<redpanda-id>]`\
+* Role: `roles/iam.workloadIdentityUser`
++
+```unset
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/rp-<redpanda-id>]"
+
+... Redpanda Console SA
+
+.... Principal: `serviceAccount:<service-project-id>.svc.id.goog[redpanda/console-<redpanda-id>]`
+
+.... Role: `roles/iam.workloadIdentityUser`
++
+The following bindings can be added with the gcloud CLI:
++
+```unset
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/console-<redpanda-id>]"
+```
+
+... Connectors SA
+
+* Principal: `serviceAccount:<service-project-id>.svc.id.goog[redpanda-connectors/connectors-<redpanda-id>]`
+* Role: `roles/iam.workloadIdentityUser`
++
+```unset
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda-connectors/connectors-<redpanda-id>]"
+```
+
+... GKE SA
```

Comment:
I might move these up to before the part where we ask the user to go to the UI and get a redpanda id.

Reply:
Please confirm that this is fine now up before step 4!